src/main/java/com/notably/view/SuggestionsWindowView.java

```diff
@@ -0,0 +1,104 @@
+package com.notably.view;
+
+import java.util.Optional;
+
+import com.notably.model.suggestion.SuggestionItem;
+
+import javafx.beans.property.Property;
+import javafx.collections.ListChangeListener;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+
+
+/**
+ * A View-Controller for the Suggestions List that appears in the View.
+ */
+public class SuggestionsWindowView extends ViewPart<Region> {
+
+    private static final String FXML = "SuggestionsWindowView.fxml";
+    private static final int LIST_CELL_HEIGHT = 28;
+
+    @FXML
+    private VBox suggestionsBox;
+
+    @FXML
+    private Label suggestionsText;
+
+    @FXML
+    private ListView<SuggestionItem> suggestionsListPanel;
+
+    public SuggestionsWindowView(ObservableList<SuggestionItem> suggestionsList,
+                                 Property<Optional<String>> responseText) {
+        super(FXML);
+        autoUpdateSuggestionsDisplay(suggestionsList, responseText);
+        initializeSuggestionsList(suggestionsList);
+    }
+
+    /**
+     * Sets listeners to update the visibility of the {@code SuggestionsWindow}, and its components.
+     * @param suggestionsList
+     * @param responseText The info text to be displayed above the suggestions list, if any.
+     */
+    private void autoUpdateSuggestionsDisplay(ObservableList<SuggestionItem> suggestionsList,
+                                              Property<Optional<String>> responseText) {
+        responseText.addListener((observable, oldValue, newValue) -> {
+            Optional<String> response = responseText.getValue();
+            if (response.isPresent() && !response.equals("")) {
```

Comment:
Actually, I believe the response text will never be empty. Do you still need to check for it?
In fact, if it's empty, we should fix the `SuggestionEngine`, rather than not displaying it at all. What do you think?

Reply:
Yup you're right, it will either not be present, or if present, not an empty string. Have refactored.